source/comply/legal-hold.rst

```diff
@@ -1,152 +1,174 @@
 
-1 - Configure your file storage as Amazon S3
+Install the plugin
+^^^^^^^^^^^^^^^^^^
+
+1. Log in to your Mattermost workspace as a system administrator.
+2. Download the latest version of the `plugin binary release <https://github.com/mattermost/mattermost-plugin-legal-hold/releases/>`_, compatible with Mattermost v8.0.1 and later. If you are using an earlier version of Mattermost, :doc:`follow our documentation </upgrade/upgrading-mattermost-server>` to upgrade to Mattermost v8.0.1 or later.
+3. Go to **System Console > Plugins > Plugin Management > Upload Plugin**, and upload the plugin binary you downloaded in the previous step.
+4. In the **Installed Plugins** section, scroll to **Legal Hold Plugin**, and select **Enable**.
+
+Configure the plugin
+^^^^^^^^^^^^^^^^^^^^^
+
+When the Legal Hold integration is enabled, you can configure when it runs using the format ``HH:MM ±HHMM`` and ``+0000`` for UTC. 
+
+You can configure a custom Amazon S3 bucket for Legal Holds by specifying Amazon S3 configuration settings. If no S3 configuration is specified, the  Mattermost server file store used. Learn more about file storage configuration options in our :ref:`product documentation <configure/environment-configuration-settings:file storage>`.
+
+(Optional) Configure a data retention policy
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. In Mattermost, go to **System Console > Environment > File Storage**.
-2. Set the **File Storage System** to **Amazon S3**.
-3. Connect the Amazon S3 storage with Mattermost by specifying the S3 bucket name, prefix, AWS region, and hostname. You may optionally specify the access keys if using IAM roles, and you can optionally specify a separate S3 Bucket than the one configured for your Mattermost server.
-4. Learn more about file storage configuration options in our :ref:`product documentation <configure/environment-configuration-settings:file storage>`.
+You can optionally configure a :doc:`data retention policy </comply/data-retention-policy>` to control how long data and file attachments are retained in the Mattermost database.
 
-.. note::
+Step 4: Create a Legal Hold
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    - For legal hold, only Amazon S3 is supported at this time, for increased reliability, compliance and automation capabilities.
-    - Support for additional file storage options such as MinIO or local file storage are under consideration.
-    - You may also optionally use our API to preserve data for legal hold. See our :ref:`electronic discovery <comply/electronic-discovery:mattermost restful api>` product documentation to learn more.
+In Mattermost, create a Legal Hold by completing the following steps:
 
-2 - Enable compliance exports
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+1. Go to **System Console > Plugins > Legal Hold Plugin**, and select **Create new**.
+2. Enter a name for the Legal Hold.
+3. Specify the user names or user groups of custodians you want to place on Legal Hold.
```

Comment:
My understanding is that a custodian is someone responsible for ensuring data is safe-guarded, not the people under investigation.